Tests/SwifQLTests/SwifQLTests.swift

```diff
@@ -315,6 +315,33 @@ final class SwifQLTests: XCTestCase {
             """)
     }
     
+    func testSelectBuilderLimitShort() {
```

Comment:
Please add it to allTests array as well (needed on Linux)

Reply:
done